src/Simplex/Chat/AppSettings.hs

```diff
@@ -43,7 +44,9 @@ data AppSettings = AppSettings
     confirmDBUpgrades :: Maybe Bool,
     androidCallOnLockScreen :: Maybe LockScreenCalls,
     iosCallKitEnabled :: Maybe Bool,
-    iosCallKitCallsInRecents :: Maybe Bool
+    iosCallKitCallsInRecents :: Maybe Bool,
+    uiColorScheme :: Maybe ColorScheme,
+    uiThemes :: Maybe [UITheme]
```

Comment:
Maybe you can include profile image corner radius in the app settings too since it's supported on every platform